goose3/configuration.py

```diff
@@ -57,7 +95,8 @@ def __init__(self):
 
         # extraction information
         self._local_storage_path = os.path.join(tempfile.gettempdir(), 'goose')
-        self._known_context_patterns = KNOWN_ARTICLE_CONTENT_PATTERNS
+        self._known_context_patterns = KNOWN_ARTICLE_CONTENT_PATTERNS[:]
```

Comment:
Fixes bug. We need to ensure we do a deepcopy here instead of just copying the mutable pointer.